src/Runner.Common/Constants.cs

```diff
@@ -129,6 +129,7 @@ public static class Flags
                     public static readonly string Ephemeral = "ephemeral";
                     public static readonly string Help = "help";
                     public static readonly string Replace = "replace";
+                    public static readonly string DisableUpdate = "disableupdate";
```

Comment:
@ethomson any conclusion for the parameter name?
https://github.com/github/c2c-actions-runtime/pull/1561#discussion_r758462142

Reply:
This is good, it matches well with the existing parameters :shipit: